README.txt

```diff
@@ -1 +1,7 @@
 https://fedorahosted.org/fedoracommunity/wiki/Development
+
+Note: Some more dependencies to be installed:-
+
+$ sudo dnf -q -y install python-webhelpers fedmsg
```

Comment:
Thanks @thunderoy.  Can you remove the ``-q`` and the ``-y`` from the README version?  The bootstrap.py script is non-interactive, which is why it has the ``-y`` option.